### ISSUE TITLE
No CPU/Memory info in with 2.0.1

### ISSUE THREAD

**Issue by dandago** [OrleansContrib/OrleansDashboard#122]
Not sure if this is a problem at all, but in the two or three quick samples I built with Orleans 2.0 and the new dashboard over the past few days, I noticed I never had memory or CPU information for the silo.

See this [screenshot](http://gigi.nullneuron.net/gigilabs/wp-content/uploads/2018/04/orleans-start-dashboard-4.png) as an example.

I was running using localhost configuration, so not sure if this or any other factors may have affected it.

I'm just raising it in case it needs further inspection.

**Comment by SebastianStehle** [comment]
CPU and Memory metrics are only enabled on Windows when you add  the Microsoft.Orleans.OrleansTelemetryConsumers.Counters package. You also have to wait some time, before you can see it.

**Comment by dandago** [comment]
Oh! Good to know!

**Comment by dandago** [comment]
Yes, though I think this should go in the readme. If it's not documented, people are going to be wondering like me.

**Comment by SebastianStehle** [comment]
Can you make a PR? ;) 

Btw: this is not specific to the dashboard, it is the orleans implementation. There is not "common" way to retrieve these information for all operation systems.

**Comment by dandago** [comment]
Sorry for the late reply - I've been really busy.

> Can you make a PR? ;)

I'm afraid I don't have time at this stage. This is a one-liner change and it would take me a lot more time just to go through all the git stuff to do this. It's a lot easier for an existing contributor.

If this remains open for a while, maybe I can give it a shot in a few weeks.

> Btw: this is not specific to the dashboard, it is the orleans implementation. There is not "common" way to retrieve these information for all operation systems.

Yes, but given that the dashboard is based on Orleans, any Orleans-related dependencies should be documented.

Otherwise there will just be a lot of people like me scratching their heads and opening issues on GitHub.
